web/consumers.py

```
@@ -0,0 +1,569 @@
+import json
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)